serverlessworkflow/modules/ROOT/pages/data-index/data-index-service.adoc

```diff
@@ -0,0 +1,225 @@
+
+The Quarkus Dev Service also allows further configuration options including:
+
+* To disable {data_index_ref} Dev Service, use the `quarkus.kogito.devservices.enabled=false` property.
+* To change the port where the {data_index_ref} Dev Service runs, use the `quarkus.kogito.devservices.port=8180` property.
+* To adjust the provisioned image, use `quarkus.kogito.devservices.imageName=quay.io/kiegroup/kogito-data-index-ephemeral` property.
+* To disable sharing the {data_index_ref} instance across multiple Quarkus applications, use `quarkus.kogito.devservices.shared=false` property.
+
+For more information about Quarkus Dev Services, see link:{dev_services_url}[Dev Services guide].
+
+[#data-index-service-configuration-properties]
+== {data_index_ref} service configuration properties
+The following table serves as a quick reference for commonly {data_index_ref} configuration properties supported.
+
+.Common configuration properties
+[cols="40%,35%,10%,10%,5%", options="header"]
+|===
+|Property|Description|Type|Default value|Override at runtime
+
+|`QUARKUS_DATASOURCE_JDBC_URL`
+| The datasource URL
+| string
+|
+| Yes
+
+|`QUARKUS_DATASOURCE_USERNAME`
+| The datasource username
+| string
+|
+| Yes
+
+|`QUARKUS_DATASOURCE_PASSWORD`
+| The datasource password
+| string
+|
+| Yes
+
+|`QUARKUS_DATASOURCE_DB_KIND`
+a|The kind of database to connect: `postgresql`,..
+|string
+|
+|Yes
+
+|`QUARKUS_FLYWAY_MIGRATE_AT_START`
+| `true` to execute Flyway automatically when the application starts, false otherwise.
+| boolean
+| false
+| Yes
+
+|`KOGITO_DATA_INDEX_QUARKUS_PROFILE`
+a| (Only when referencing an image distribution)
+
+Allows to change the event connection type. The possible values are:
+
+* `kafka-events-support`
+* `http-events-support`
+
+| string
+| `kafka-events-support`
+| Yes
+
+|`kogito.persistence.query.timeout.millis`
+|Defines timeout for a query execution.
+|long
+|`10000`
+|Yes
+
+|`quarkus.kogito.devservices.enabled`
+|Enables or disables the Dev Services for workflows. By default, the Dev Services are enabled, unless an existing configuration is present.
+|boolean
+|`true`
+|No
+
+|`quarkus.kogito.devservices.port`
+|Defines the optional fixed port that the Dev Services listen to.
+|int
+|`8180`
+|No
+
+|`quarkus.kogito.devservices.image-name`
+|Defines the {data_index_ref} image to use in Dev Service.
+|string
+|`quay.io/kiegroup/kogito-data-index-ephemeral:{page-component-version}`
+|No
+
+|`quarkus.kogito.devservices.shared`
+|Indicates if the {data_index_ref} instance, which is managed by Quarkus Dev Services, is shared.
+|boolean
+|`true`
+|No
+
+|`quarkus.kogito.devservices.service-name`
+|Defines the value of the label that is attached to the started container. This property is used when `shared` is set to `true`.
+|string
+|`kogito-data-index`
+|No
+
+
+|===
+
+== Additional resources
+
+* xref:core/timeouts-support.adoc[Timeouts in {product_name}]
```

Comment:
Timeouts again ? :)

Reply:
ups!. I've added some new references to eventing and data index core concepts